test/mpi/coll/test_coll_algos.sh

```diff
@@ -27,7 +27,7 @@ for algo_name in ${algo_names}; do
 
         coll_algo_tests+="bcasttest 10 ${env}${nl}"
         coll_algo_tests+="bcastzerotype 5 ${env}${nl}"
-        coll_algo_tests+="bcasttest 10 ${env}${nl}"
+        coll_algo_tests+="bcasttest 10 ${env} env=MPIR_CVAR_IBCAST_SCATTER_KVAL=${kval} env=MPIR_CVAR_IBCAST_ALLGATHER_RECEXCH_KVAL=${kval} ${nl}"
```

Comment:
So we're duplicating this test now? This addition seems weird.

Reply:
Fixed.